compiler/src/iree/compiler/Codegen/Interfaces/ProcessorOpInterfaces.cpp

```diff
@@ -57,7 +57,7 @@ struct WorkgroupIdOpInterface
 
 struct WorkgroupCountOpInterface
     : public ProcessorCountInterface::ExternalModel<
-          WorkgroupCountOpInterface, IREE::HAL::InterfaceWorkgroupIDOp> {
+          WorkgroupCountOpInterface, IREE::HAL::InterfaceWorkgroupCountOp> {
```

Comment:
This and below seem to have been bugs, if there were no assertions triggered by casts to the wrong op type in the respecitve `getDimIndex` implementations, this may indicate a lack of test coverage.